utils/utils.go

```diff
@@ -19,13 +19,15 @@
 package utils
 
 import (
+	"os"
 	"strings"
 
 	k8sUtils "github.com/BLasan/APKCTL-Demo/k8s"
 )
 
 func GetAPKCTLHomeDir() string {
-	return "/Users/benura/Desktop/APKCTL"
+	pwd, _ := os.Getwd()
```

Comment:
Shouldn't we handle the error returned here?

Reply:
Will fix with the next PR